src/Model/Common/Role.java

```diff
@@ -0,0 +1,147 @@
+package Model.Common;
+
+import Model.Conection.DataAccess;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * @autor: JuanaRodriguez
+ */
+public class Role {
+    private int roleId;
+    private String roleName;
+    private DataAccess dbAccess;
+
+    /**
+     * Give back a role
```

Comment:
give back a role?? is this correct because the method returns the Identifier of the role